sidik/basic_11.go

```diff
@@ -8,7 +8,7 @@ import (
 var ( //membuat variabel bisa sekaligus dengan pembungkus
 	Tobe bool	= false
 	Maxint uint64 = 1<<64 - 1
-	zo	complex128 = cmplx.Sqrt(-5 + 12i)
+	zo	complex128 = cmplx.Sqrt(-5 + 12i) //type data complex128
```

Comment:
belum tau type data complex128

Reply:
**Complex Numbers**: The complex numbers are divided into two parts are shown in the below table. float32 and float64 are also part of these complex numbers. The in-built function creates a complex number from its imaginary and real part and in-built imaginary and real function extract those parts.

https://www.geeksforgeeks.org/data-types-in-go/

Reply:
oke